index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+export default function(...classes: ClassValue[]): string;
```

Comment:
@adidahiya Would you recommend updating the DefinitelyTyped definition to remove it as well? Or should we just leave it out in this PR?